dotCMS/src/main/java/com/dotcms/storage/StorageProvider.java

```diff
@@ -0,0 +1,39 @@
+package com.dotcms.storage;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class StorageProvider {
+
+    private final Map<String, Storage> fileSystemStorageMap = new ConcurrentHashMap<>();
```

Comment:
maybe a generic name? because it stores multiple storage types and not only "fileStorage"